### PR TITLE
swap program

### PR DESCRIPTION
**Pull request by thakurrohithsingh**

### CODE REVIEW

File: swap.c [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+void swap(int *var1,int *var2)
+{
+    int tmp;
+    tmp = *var1;
```
Comment (Abhivardhan) [inline swap.c]: There is no point using pointers here when your intention is to swap the values. Either swap the addresses or remove the pointers

File: swap.c [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+void swap(int *var1,int *var2)
+{
+    int tmp;
+    tmp = *var1; 
+    *var1=*var2;    
+    *var2=tmp; 
+    printf("%d %d",*var1,*var2);
+}
+int main(){
+    int var1,var2;
```
Comment (Abhivardhan) [inline swap.c]: Naming the variables as var1, var2 is equal to naming them as x and y. It doesnt convey any meaning. rename properly

File: swap.c (hunk 1 above)
Comment (thakurrohithsingh) [reply swap.c]: ok sir

File: swap.c (hunk 2 above)
Comment (thakurrohithsingh) [reply swap.c]: should I change it to x and y

Comment (Abhivardhan) [reply swap.c]: changes it to num1, num2

Comment (thakurrohithsingh) [reply swap.c]: ok sir

**Pull request merged**